database/migrations/2022_10_19_013449_create_games_table.php

```diff
@@ -19,6 +19,7 @@ public function up()
             $table->decimal('complexity_rating', 3, 2);
             $table->year('release_year');
             $table->foreignId('publisher_id')->constrained();
+            $table->foreignId('category_id')->constrained();
```

Comment:
we don't want this, as there is a many-to-many relationship between games and categories

Reply:
I'm struggling with getting the seeder to work - it says it has an incorrect integer value so it won't run. Is this because I need the PublisherSeeder code?

Also, I'm assuming if I remove the category_id from the Games table, it'll use the category_game table to link the games to the category? Should I add an foreign key to the category_game table instead?

Reply:
yeah i just merged Laura's PR so publisher should work now. and the game_category table already has foreign keys, nothing else is needed.